runtime.c

```diff
@@ -385,6 +392,38 @@ main (int argc, char *argv[])
         exit(0);
     }
 
+    if (arg && strcmp(arg,"appimage-portable-home")==0) {
```

Comment:
Please add spaces between the arguments in `strcmp` and around the `==` to improve readability.

Reply:
I used exactly the same style used in the rest. I also don't like not having spaces, but I prefer consistency over personal preference. I actually think we should run clang-format or some other tool to format the whole file.

Reply:
>  I actually think we should run clang-format or some other tool to format the whole file.

Agree. I think we also need to pick one of the common C styles.